### PR TITLE
Jose5 actions

### PR DESCRIPTION
**Pull request by jmgiron98**
Created a version of the deep q_learner that will learn on all five actions and testing a reward function which rewards the positive motion of Forest's knees.

### CODE REVIEW

File: controller.py [hunk 1]
```diff
@@ -33,4 +33,4 @@ def run(self):
 
 if __name__ == '__main__':
     cntrlr = Controller()
-    cntrlr.run()
\ No newline at end of file
+    cntrlr.run()
```
Comment (N-Demir) [inline controller.py]: I can't believe you've done this

Comment (jmgiron98) [reply controller.py]: Jajaja computer did it for me when I saved

Comment (jmgiron98) [reply controller.py]: Tried to delete it but everytime I saved, it just added it again so I left it.

Comment (N-Demir) [reply controller.py]: Damn you!!!

**Pull request merged**